big-a/package-lock.json

```diff
@@ -27,7 +27,6 @@
                 "classnames": "^2.5.1",
                 "core-js": "^3.36.1",
                 "joi": "^17.12.1",
-                "js-yaml": "^4.1.0",
```

Comment:
remove this dependency, since we are not actually using this. This takes up additional space without adding functionality